mlflow/values.yaml

```diff
@@ -1,12 +1,15 @@
-backend_store_uri: "file:///mlruns"
-default_artifact_root: ""
+# TODO: set good defaults and override for GCP.
+backend_store_uri: "sqlite:///mlruns.sqlite"
```

Comment:
This has a couple problems. This would break when we deploy to prod, since both would be writing to the same database file.

I should figure out how persistent volumes and PVCs work.